File: command/misc/ping.py
import discord 
from function.Text import *

class Ping():

    def __init__(self, message, bot):
        self.message = message
        self.channel = message.channel
        self.bot = bot
    
    async def run(self):
        import time
        now = time.monotonic()
        msg = await self.channel.send(f'{str(Text("fr", "Ping").get())}')
        ping = int((time.monotonic() - now) * 1000)
        await msg.edit(content=f'{str(Text("fr", "Ping").get())} ``{ping} ms``')

File: command/misc/ping.js

const Discord = require("discord.js"); // Voir index.js

exports.run = async (message, client, args, lang) => { // Remplace def __init__ / class Ping()
        var now = Date.now(); // Permet d'avoir la date en ms (timestamp), équivalent du time.monotonic()
        message.channel.send(lang.ping.latency).then((msg) => { // Envoies le message et le garde pour plus tard
            var after = Date.now() - now; // Calcul le ping
            msg.edit(`${lang.ping.latency} ${after} ms`); // Edit le message
        })
}